silo-core/test/foundry/utils/hook-receivers/liquidation/LiquidationHelper_1token.i.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.28;
+
+import {IERC20} from "openzeppelin5/token/ERC20/IERC20.sol";
+
+import {ISiloConfig} from "silo-core/contracts/interfaces/ISiloConfig.sol";
+
+import {LiquidationHelperCommon} from "./LiquidationHelperCommon.sol";
+
+/*
+    FOUNDRY_PROFILE=core-test forge test -vv --ffi --mc LiquidationHelper1TokenTest
+*/
+contract LiquidationHelper1TokenTest is LiquidationHelperCommon {
```

Comment:
Do we want to add an integration test with already deployed silos on Arbitrum? I think it is must-have.

Reply:
what's the difference between Silo in test and Silo deployed on Arbitrum? I think there is no difference. And as I mention on standup, there is TODO to add at least one test with real swap, but vaults has higher priority, so this will be addresses most likely in separate PR

Reply:
There is no difference between silos. Seeing that it works in a production environment makes me more confident about the codebase. For example, swaps functionality. The code looks "ok", but it is not tested. I'm not sure if it will work or not. See the test that executes swaps would answer all questions.